src/engine/enginemaster.cpp

```diff
@@ -155,14 +171,17 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
     m_pKeylockEngine->set(pConfig->getValueString(
             ConfigKey(group, "keylock_engine")).toDouble());
 
+    // TODO: Make this read only but let EngineMasterTest enable it
```

Comment:
Than the TODO needs to be rewritten:
// TODO: Make this read only and let the Engine decide if it needs a master mix or not.